reference/performance/statement-summary.md

```diff
@@ -24,45 +26,145 @@ select * from EMPLOYEE where ID in (4, 5) and SALARY between 3000 and 4000;
+>
+> In TiDB, the time unit of fields in statement summary tables is nanosecond (ns), whereas in MySQL it is picosecond (ps).
+
+### Fields description
+
+Basic fields:
+
+- `STMT_TYPE`: SQL statement type.
+- `SCHEMA_NAME`: The current schema under which SQL statements of this category are executed.
+- `DIGEST`: The digest of SQL statements of this category.
+- `DIGEST_TEXT`: Normalized SQL statement.
+- `QUERY_SAMPLE_TEXT`: One of the original SQL statements where SQL statements of this category appeared.
+- `TABLE_NAMES`: All tables involved in SQL statements. If there is more than one table, each is separated by a comma.
+- `INDEX_NAMES`: All SQL indexes used in SQL statements. If there is more than one index, each is separated by a comma.
+- `SAMPLE_USER`: One of the users who execute SQL statements of this category.
+- `PLAN_DIGEST`: Digest of the execution plan.
+- `PLAN`: The original execution plan. If there is more than one statement, display only the execution plan of one statement.
+
+Execution time related fields:
+
+- `SUMMARY_BEGIN_TIME`: The begin time of the current statistical period.
+- `SUMMARY_END_TIME`: The end time of the current statistical period.
+- `FIRST_SEEN`: The time when SQL statements of this category are seen for the first time.
+- `LAST_SEEN`: The time when SQL statements of this category are seen for the last time.
+
+TiDB server related fields:
+
+- `EXEC_COUNT`: Total execution times of SQL statements of this category.
+- `SUM_LATENCY`: Total execution latency of SQL statements of this category.
+- `MAX_LATENCY`: Maximum latency of SQL statements of this category.
+- `MIN_LATENCY`: Minimum latency of SQL statements of this category.
+- `AVG_LATENCY`: Average latency of SQL statements of this category.
+- `AVG_PARSE_LATENCY`: Average latency of the parser.
+- `MAX_PARSE_LATENCY`: Maximum latency of the parser.
+- `AVG_COMPILE_LATENCY`: Average latency of the compiler.
+- `MAX_COMPILE_LATENCY`: Maximum latency of the compiler.
+- `AVG_MEM`: Average memory (byte) used.
+- `MAX_MEM`: Maximum memory (byte) used.
+
+TiKV Coprocessor task related fields:
+
+- `COP_TASK_NUM`: The number of Coprocessor requests that a SQL statement sends.
+- `AVG_COP_PROCESS_TIME`: The average execution time of Coprocessor tasks.
+- `MAX_COP_PROCESS_TIME`: The maximum execution time of Coprocessor tasks.
+- `MAX_COP_PROCESS_ADDRESS`: The address of the Coprocessor task with the longest execution time.
+- `AVG_COP_WAIT_TIME`: The average waiting time of Coprocessor tasks.
+- `MAX_COP_WAIT_TIME`: The maximum waiting time of Coprocessor tasks.
+- `MAX_COP_WAIT_ADDRESS`: The address of the Coprocessor task with the longest waiting time.
+- `AVG_PROCESS_TIME`: The average processing time of SQL statements in TiKV.
+- `MAX_PROCESS_TIME`: The maximum processing time of SQL statements in TiKV.
+- `AVG_WAIT_TIME`: The average waiting time of SQL statements in TiKV.
+- `MAX_WAIT_TIME`: The maximum waiting time of SQL statements in TiKV.
```

Comment:
Please confirm with the author whether it refers to `a SQL statement` or `SQL statements`.

Reply:
@TomShawn I double-checked with the author. It refers to `SQL statements`.

Reply:
It's OK. It refers to `SQL statements`.